Studio-Lovelies/GG-JointJustice-Unity
# Enter Konami Code to play baby scene

## Summary
Enter the Konami Code (Up, Up, Down, Down, Left, Right, Left, Right, B, A, Start/Enter) to play the `baby` scene.

## Testing instructions
1. Play the main menu scene and press enter to start a new game
2. Notice the game starting Case 1-1
3. Restart the main menu scene enter the Konami Code with the final press of "enter" to start a new game
4. Notice the game starting the baby scene

## Additional information

- `[ ]` Changes UI
- `[x]` Introduces new feature
- `[ ]` Removes existing feature
- `[ ]` Has associated resource:
